core/state_transition.go

```diff
@@ -337,7 +337,13 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		effectiveTip = cmath.BigMin(st.gasTipCap, new(big.Int).Sub(st.gasFeeCap, st.evm.Context.BaseFee))
 	}
 	amount := new(big.Int).Mul(new(big.Int).SetUint64(st.gasUsed()), effectiveTip)
-	st.state.AddBalance(st.evm.Context.Coinbase, amount)
+	if london {
+		governanceAddress := common.HexToAddress(st.evm.ChainConfig().Bor.CalculateGovernanceContract(st.evm.Context.BlockNumber.Uint64()))
+		st.state.AddBalance(governanceAddress, amount)
+	} else {
+		st.state.AddBalance(st.evm.Context.Coinbase, amount)
```

Comment:
This should be in `else`. We want to transfer `effectiveTip` to Coinbase and other amount (which is burning in Ethereum mainnet to address)

Also please do not call it a governance address since it is not a governance address. It's `burnContractAddress`.